app/gateway/2.6.x/install-and-run/upgrade.md

```diff
@@ -3,11 +3,11 @@ title: Upgrade Guide
 toc: true
 ---
 
-Upgrade to major, minor, and patch {{site.ee_product_name}} releases using the
+Upgrade to major, minor, and patch Kong Gateway releases using the
 `kong migrations` commands.
 
 You can also use the commands to migrate all Kong Gateway OSS entities to Kong Gateway Enterprise. See
-[Migrating from Kong Gateway to Kong Enterprise](/enterprise/{{page.kong_version}}/deployment/upgrades/migrate-ce-to-ke/).
+[Migrating from Kong Gateway OSS to Kong Gateway Enterprise](/upgrades/migrate-ce-to-ke/).
```

Comment:
```suggestion
[Migrating from Kong Gateway OSS to Kong Gateway Enterprise](https://docs.konghq.com/upgrades/migrate-ce-to-ke/).
```
Let's use absolute URLs so that they also work in the code repo (applies to all relative URLs in this topic)

Will have to ask if it's fine to include this line about upgrading from OSS to Enterprise, or we need to leave it out entirely.